### ISSUE TITLE
Show SRPM url

### ISSUE THREAD

**Issue by lachmanfrantisek** [packit/dashboard#70]
- [ ] Add a new column to the `SRPM Builds` tab [here](https://dashboard.packit.dev/jobs). (Or find a better place where we can put this info.)

Requires: https://github.com/packit/packit-service/issues/958

**Comment by mfocko** [comment]
could be probably included in packit/packit-service#912

**Comment by lachmanfrantisek** [comment]
@mfocko I don't so -- it's about date-time and result page, not a dashboard.

**Comment by mfocko** [comment]
> @mfocko I don't so -- it's about date-time and result page, not a dashboard.

yeah I know, but we could include more info in the SRPM logs page apart from logs, e.g. commit sha, said date-time or also the URL

from commit status we can get to build result and from there to srpm logs and COPR or SRPM URL
however from dashboard we can get **only directly** to COPR or SRPM **logs**, so adding more info to either the page with SRPM logs or dashboard would be beneficial

**Comment by lachmanfrantisek** [comment]
Since we want to replace result-page by the dashboard, I would leave this issue as is (here).

But you have a good point, there is other information missing on the dashboard site. Although I expected this to be done when working on the internal SPIKE card, we can create a new issue that will define some dashboard alternative to the current result page(s). (The table for all results will not be enough for all the information. I have some detail single-result view in mind.)

(Once that other issue is implemented, this one won't be needed probably. But this is much quicker to do in the meantime.)

**Comment by IceWreck** [comment]
> Since we want to replace result-page by the dashboard, I would leave this issue as is (here).

i don't have the local dev environment for packit service setup right now but if you can make that information available via the API, I'll add it to the dashboard

**Comment by lachmanfrantisek** [comment]
@IceWreck That would be really nice!

Sure, we can. I'll describe at least some basics in a new issue so you can know what to expect.

**Comment by lachmanfrantisek** [comment]
> I'll describe at least some basics in a new issue so you can know what to expect.

Here it is: #71

**Comment by lachmanfrantisek** [comment]
The API already contains the URL value. (example: https://stg.packit.dev/api/srpm-builds/3000)

**Comment by mfocko** [comment]
Resolved by 5c0bb706214e34c660db28f0b703f9d0f3d7ec9a, it is kept in result page
@lachmanfrantisek do we still want the column in the table or are we fine with the current state?

**Comment by lachmanfrantisek** [comment]
@mfocko You are right, we can maybe improve formatting, but good enough for now:

![Screenshot from 2021-03-11 16-53-52](https://user-images.githubusercontent.com/20214043/110815161-7f6ed600-828a-11eb-9008-e7544a302dc2.png)
